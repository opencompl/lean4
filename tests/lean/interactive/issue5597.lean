example : True := by
  simp!
 --^ textDocument/hover
