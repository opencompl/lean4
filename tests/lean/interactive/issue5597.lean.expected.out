{"textDocument": {"uri": "file:///issue5597.lean"},
 "position": {"line": 1, "character": 3}}
{"range":
 {"start": {"line": 1, "character": 2}, "end": {"line": 1, "character": 7}},
 "contents":
 {"value":
  "`simp!` is shorthand for `simp` with `autoUnfold := true`.\nThis will rewrite with all equation lemmas, which can be used to\npartially evaluate many definitions. ",
  "kind": "markdown"}}
